hotel/models/hotel_folio.py

```diff
@@ -409,11 +409,11 @@ def action_invoice_create(self, grouped=False, final=False):
         """
         # fixme does not seem to go through here
         room_lst = []
-        invoice_id = self.order_id.action_invoice_create(
+        invoice_id = self.mapped("order_id").action_invoice_create(
             grouped=False, final=False
         )
         for line in self:
-            values = {"invoiced": True, "hotel_invoice_id": invoice_id}
+            values = {"invoiced": True, "hotel_invoice_id": invoice_id[0]}
```

Comment:
Isn't `self` a folio here?

Reply:
Thanks for the review!

`self` is a recordset of folio's (see `@api.multi`). The point o this PR is that there can be multiple folios in this recordset.

Seeing this again, I now think that `line` could have a better name such as `folio`, and that `invoice_id` could be renamed `invoice_list` (the output of `action_invoice_create()` on sale orders is a list of invoices).

Could you clarify what you were suggesting with your remark?

Reply:
It was about better naming for the loop :-)